From b5d6202cfd88fc7a4192dd91e20900a4a99f4211 Mon Sep 17 00:00:00 2001
From: Eli Billauer <eli.billauer@gmail.com>
Date: Wed, 19 Feb 2014 02:02:21 +0200
Subject: [PATCH 3/8] usb/core/hub.c: Export usb_port_suspend(), because phy-zynq-usb.c needs it


Signed-off-by: Eli Billauer <eli.billauer@gmail.com>
---
 drivers/usb/core/hub.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/drivers/usb/core/hub.c b/drivers/usb/core/hub.c
index 744f524..7841ff8 100644
--- a/drivers/usb/core/hub.c
+++ b/drivers/usb/core/hub.c
@@ -3074,6 +3074,8 @@ int usb_port_suspend(struct usb_device *udev, pm_message_t msg)
 	return status;
 }
 
+EXPORT_SYMBOL_GPL(usb_port_suspend);
+
 /*
  * If the USB "suspend" state is in use (rather than "global suspend"),
  * many devices will be individually taken out of suspend state using
-- 
1.7.2.3

